lib/shoulda/matchers/active_model/numericality_matchers/even_number_matcher.rb

```diff
@@ -20,6 +20,11 @@ def allowed_type
           def diff_to_compare
             2
           end
+
+          def strict
```

Comment:
Actually, I just checked and this is already contained in ValidationMatcher so you can get rid of this. The only thing you need is the loop in ValidateNumericalityOfMatcher to set `strict` on each submatcher.